tck/features/OrderByAcceptance.feature

```diff
@@ -0,0 +1,151 @@
+#
+# Copyright 2016 "Neo Technology",
+# Network Engine for Objects in Lund AB (http://neotechnology.com)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+Feature: OrderByAcceptance
+
+  Background:
+    Given any graph
+
+  Scenario: ORDER BY should return results in ascending order
+    And having executed: CREATE (n1 {prop: 1}), (n2 {prop: 3}), (n3 {prop: -5})
+    When executing query: MATCH (n) RETURN n.prop AS prop ORDER BY n.prop
+    Then the result should be:
+      | prop  |
+      | -5    |
+      | 1     |
+      | 3     |
+
+
+  Scenario: ORDER BY DESC should return results in descending order
+    And having executed: CREATE (n1 {prop: 1}), (n2 {prop: 3}), (n3 {prop: -5})
+    When executing query: MATCH (n) RETURN n.prop AS prop ORDER BY n.prop
+    Then the result should be:
+      | prop  |
+      | 3     |
+      | 1     |
+      | -5    |
+
+  Scenario: ORDER BY of a column introduced in RETURN should return salient results in ascending order
+    When executing query: WITH [0, 1] AS prows, [[2], [3, 4]] AS qrows UNWIND prows AS p UNWIND qrows[p] AS q WITH p, count(q) AS rng RETURN p ORDER BY rng
+    Then the result should be:
+      | p  |
+      | 0  |
+      | 1  |
+
+  Scenario: renaming columns before ORDER BY should return results in ascending order
+    And having executed: CREATE (n1 {prop: 1}), (n2 {prop: 3}), (n3 {prop: -5})
+    When executing query: MATCH (n) RETURN n.prop AS n ORDER BY n + 2
+    Then the result should be:
+      | n   |
+      | -5  |
+      | 1   |
+      | 3   |
+
+  Scenario: handle projections with ORDER BY (GH#4937)
+    And having executed: CREATE (c1:Crew {name: 'Neo', rank: 1}), (c2:Crew {name: 'Neo', rank: 2}), (c3:Crew {name: 'Neo', rank: 3}), (c4:Crew {name: 'Neo', rank: 4}), (c5:Crew {name: 'Neo', rank: 5})
+    When executing query: MATCH (c:Crew {name: 'Neo'}) WITH c, 0 AS relevance RETURN c.rank AS rank ORDER BY relevance, c.rank
+    Then the result should be:
+      | rank  |
```

Comment:
Many of the results tables here have one space too much in the inner right padding.